src/app/events/[slug]/components/SponsorContainer.tsx

```diff
@@ -3,21 +3,26 @@ import { clsx } from 'clsx'
 import { type SponsorLogoProps, SponsorLogo } from './SponsorLogo'
 import type { Sponsor } from './SponsorSection'
 
-type SponsorGridProps = {
+type SponsorContainerProps = {
   sponsors: Array<Sponsor>
   tier: string
-  gridClassName: string
   logoImageConfig: SponsorLogoProps['logoImageConfig']
+  gap?: string
```

Comment:
Is there a reason for the `gap` to be optional? It seems like something we want to control.